generators/app/index.js

```diff
@@ -159,7 +159,7 @@ module.exports = class extends Generator {
                 clearCoverage: "shx rm -rf coverage",
                 karma: "run-s clearCoverage karma-ci",
                 lint: "eslint ./**/webapp/**/*.js && prettier --check ./**/webapp/**/*.{js,xml}",
-                "lint-fix": "eslint ./**/webapp/**/*.js --fix && prettier --write ./**/webapp/**/*.{js,xml}"
+                "lint-fix": "eslint ./**/webapp/**/*.js --fix && prettier --write ./**/webapp/**/*.{js,xml} --no-error-on-unmatched-pattern"
```

Comment:
Hmm, but here the parameter looks like being assigned to prettier - was the issue a prettier issue?

Reply:
No, this was the first commit regarding the error message for the linting of JSON Views. Check out the other commit.